src/BannerWithLink/styles.ts

```diff
@@ -1,9 +1,14 @@
 import { css } from '@emotion/react';
-import { from, until, body, neutral, space } from '@guardian/source-foundations';
+import { from, until, space, neutral } from '@guardian/source-foundations';
+import { ButtonTheme, buttonThemeReaderRevenueBrandAlt } from '@guardian/source-react-components';
 
-const bodyColor = '#666';
+const backgroundColor = '#f79e1b';
```

Comment:
I wonder if we should name this component OrangeBannerWithLink so that any campaigns that want to use this orange background don't have to be updated if BannerWithLink wants to be used for a different coloured campaign in the future.

Reply:
Good question! I'm tempted to leave this as-is for now while we wait and see how we want to handle palettes in banners. I think it's probably low risk in terms of having to update canvases as this look back "moment" will have a finite lifespan.